tests-e2e/cypress/e2e/user-activation/activateEdxUserAccount.cy.ts

```diff
@@ -120,11 +119,8 @@ describe('Activate EDX User Account Page', () => {
             cy.get(selectors.userActivationPage.personalActivationCodeInput).type(personalCode);
           });
           cy.get(selectors.userActivationPage.userActivationSubmitButton).click();
-          cy.wait(11000);
-          cy.get(selectors.userActivationPage.userActivationSnackBar).should('include.text',
-              'User could activate account successfully');
-          // cy.url().should('eq', 'https://dev.educationdataexchange.gov.bc.ca/institute-selection'
-
+          cy.get(selectors.userActivationPage.userActivationSnackBar, {timeout:9000}).should('include.text',
```

Comment:
We have 8000 in the previous hunk and 9000ms in this one. Could we please use the lowest possible timeout for this exception, please?